lib/chainsync/chainsync/db/hyperdrive/interface.py

```diff
@@ -423,8 +478,8 @@ def get_wallet_deltas(
     return pd.read_sql(query.statement, con=session.connection(), coerce_float=coerce_float)
 
 
-def get_agents(session: Session, start_block: int | None = None, end_block: int | None = None) -> list[str]:
-    """Get the list of all agents from the WalletInfo table.
+def get_all_traders(session: Session, start_block: int | None = None, end_block: int | None = None) -> list[str]:
```

Comment:
the docstring still says "agent addresses". But tbh, I think a more appropriate name would be `get_all_addresses` since there seems to be no distinction in the code about whether the address belongs to a bot ("agent") or human (what I assume you mean by "trader")

Reply:
This function is supposed to be getting all addresses that have made a trade, regardless of bot or human. This information is gathered from transactions themselves

Reply:
Hence, "trader"